_posts/en/newsletters/2024-02-14-newsletter.md

```diff
@@ -0,0 +1,283 @@
+  a5link="https://bitcoincore.reviews/28950#l-108"
+
+  q6="Why can’t `maxfeerate` be checked immediately after decoding like
+      `maxburnamount` is?"
+  a6="Because transaction inputs famously don't explicitly state the
+      input amount; they can only be known after looking up the parent
+      output. The fee rate requires the fee, which requires the input
+      amounts."
+  a6link="https://bitcoincore.reviews/28950#l-141"
+
+  q7="How does the `maxfeerate` check in `testmempoolaccept` RPC differ
+      from `submitpackage` RPC? Why can’t they be the same?"
+  a7="`submitpackage` uses modified fees while
+      `testmempoolaccept` uses base fees, as explained earlier.
+      Also, the fee rate check is done after the `testaccept` package
+      processing because the transactions are not added
+      to mempool and broadcasted after the processing, so we can safely
+      check `maxfeerate` and return appropriate error messages.
+      The same can't be done in `submitpackage` because the package
+      transactions might have already been accepted into the mempool and
+      broadcasted to peers, rendering the check redundant."
+  a7link="https://bitcoincore.reviews/28950#l-153"
+%}
+
+## Releases and release candidates
+
+*New releases and release candidates for popular Bitcoin infrastructure
+projects.  Please consider upgrading to new releases or helping to test
+release candidates.*
+
+<!-- FIXME: harding to update Tuesday -->
+
+## Notable code and documentation changes
+
+_Notable recent changes in [Bitcoin Core][bitcoin core repo], [Core
+Lightning][core lightning repo], [Eclair][eclair repo], [LDK][ldk repo],
+[LND][lnd repo], [libsecp256k1][libsecp256k1 repo], [Hardware Wallet
+Interface (HWI)][hwi repo], [Rust Bitcoin][rust bitcoin repo], [BTCPay
+Server][btcpay server repo], [BDK][bdk repo], [Bitcoin Improvement
+Proposals (BIPs)][bips repo], [Lightning BOLTs][bolts repo],
+[Bitcoin Inquisition][bitcoin inquisition repo], and [BINANAs][binana
+repo]._
+
+- [Bitcoin Core #28948][] adds support for [version 3 transaction
+  relay][topic v3 transaction relay] allowing any v3 transaction that
+  has no unconfirmed parent to enter the mempool according to the normal
+  transaction acceptance rules.  The v3 transaction can be [CPFP fee
+  bumped ][topic cpfp] but only if the child is 1,000 vbytes or less.
+  Each v3 parent may only have one unconfirmed child transaction in the
+  mempool and each child may only have one unconfirmed parent.  Either
+  the parent or child transaction can always be [replaced by fee][topic
+  rbf].  The rules only apply to Bitcoin Core's relay policy; at the
+  consensus layer, v3 transactions are validated the same as version 2
+  transactions defined in [BIP68][].  The new rules are intended to help
+  contract protocols such as LN ensure their precommitted transactions
+  can always be confirmed quickly with minimal extra fees needed to
+  escape [transaction pinning][topic transaction pinning].
```

Comment:
```suggestion
  escape [transaction pinning][topic transaction pinning] attacks.
```

Reply:
Leaving as-is.  There are legit cases where transaction pinning can happen by accident, especially if CPFP carve out wasn't a thing (which it might not be sooner or later), so pinning isn't always an attack.